test/controller.test.ts

```diff
@@ -2052,6 +2567,17 @@ contract('Controller', ([owner, accountOwner1, accountOperator1, random]) => {
 
   describe('Expiry', () => {
     it('should return false for non expired otoken', async () => {
+      const otoken: MockOtokenInstance = await MockOtoken.new()
+      await otoken.init(
+        addressBook.address,
+        weth.address,
+        usdc.address,
+        usdc.address,
+        new BigNumber(200).times(new BigNumber(10).exponentiatedBy(18)),
+        new BigNumber(await time.latest()).plus(60000 * 60000),
+        true,
+      )
+
       assert.equal(await controller.isExpired(otoken.address), false, 'Otoken expiry check mismatch')
     })
 
```

Comment:
Can you add a test to be able to exercise many series of oTokens in 1 transaction?

Reply:
added.